connectors/src/connectors/github/lib/utils.ts

```diff
@@ -65,3 +67,54 @@ export function matchGithubInternalIdType(internalId: string): {
   }
   throw new Error(`Invalid Github internal id: ${internalId}`);
 }
+
+export function getRepositoryInternalId(repoId: string | number): string {
+  return `github-repository-${repoId}`;
+}
+
+export function getIssuesInternalId(repoId: string | number): string {
+  return `github-issues-${repoId}`;
+}
+
+export function getIssueDocumentId(
```

Comment:
the mix between internal id and document id naming here is a bit confusing, especially since the initial phrasing of the issue is to standardize "from internal id to document id" while here they seem to all be "good ids"

What about avoiding those names, eg. everybody called get...NodeId?

Reply:
I initially used internalId everywhere but changed as per comment above, I'm all in for calling them nodeId!

Reply:
ok it's a nit so fine to leave as is

Reply:
(do as you think best)

Reply:
went for nodeId everywhere 👍